src/sentry/runner/commands/configoptions.py

```diff
@@ -166,22 +158,34 @@ def patch(ctx) -> None:
     from sentry.utils import metrics
 
     dry_run = bool(ctx.obj["dry_run"])
+    presenter_delegator = ctx.obj["presenter_delegator"]
     if dry_run:
         click.echo("!!! Dry-run flag on. No update will be performed.")
 
     for key, value in ctx.obj["options_to_update"].items():
         try:
             _attempt_update(
-                key, value, ctx.obj["drifted_options"], dry_run, bool(ctx.obj["hide_drift"])
+                presenter_delegator,
+                key,
+                value,
+                ctx.obj["drifted_options"],
+                dry_run,
+                bool(ctx.obj["hide_drift"]),
             )
         except Exception:
             metrics.incr(
                 "options_automator.run",
                 tags={"status": "update_failed"},
                 sample_rate=1.0,
             )
+            presenter_delegator.error(
+                key, "Updated failed. Check db connection, option type, and option spelling."
```

Comment:
That is not a useful error message. 
Since we are not reporting the error just via stacktrace you will have to distinguish different types of errors and provide proper messages. 
- If the option does not exists, you will receive `UnknownOption` 
- if the type is wrong it is something like `TypeError`